java/client-test/src/test/java/no/bankaxept/epayment/test/client/base/AccessTokenPublisherTest.java

```diff
@@ -56,7 +59,7 @@ public void should_schedule_on_startup_then_new_on_success() throws IOException
         accessTokenProcessor = new ScheduledAccessTokenPublisher("uri", "key", "username", "password", clock, schedulerMock, httpClientMock);
         accessTokenProcessor.subscribe(subscriberMock);
         verify(schedulerMock).schedule(any(Runnable.class), eq(0L), eq(TimeUnit.MILLISECONDS));
-        verify(schedulerMock, Mockito.after(2000)).schedule(any(Runnable.class), eq(599999L), eq(TimeUnit.MILLISECONDS));
+        verify(schedulerMock, Mockito.after(2000)).schedule(any(Runnable.class), and(gt(1189000L), lt(1190000L)) , eq(TimeUnit.MILLISECONDS));
```

Comment:
was a pain to get "rougly this time".. But this *works* at least. Not sure if my precision is too loose